### DISCUSSION TITLE
Cross Processing

### DISCUSSION THREAD

**Discussion by UkooLabs** [SixLabors/ImageSharp#1361] [Ideas]
I have a need to create the Cross Processing effect (X-Pro) I assume Image sharp currently does not have it.

So currently looking to reverse engineer the filter from https://www.canva.com/ and port to C# if even possible

**Comment by JimBobSquarePants** [comment]
It doesn't have it no. 

From memory I would imagine the effect could be achieved with a combination of color matrix adjustments and white balance. 

I wonder whether implementing a white balance adjustment mechanism would be sufficient?

I'd have to see an example of output to be confident though.
